Packs/PaloAltoNetworks_Threat_Vault/Integrations/Threat_Vault/Threat_Vault.yml

```diff
@@ -0,0 +1,356 @@
+      type: String
+    - contextPath: ThreatVault.Search.status
+      description: Saearch status.
+      type: String
+  - arguments:
+    - default: false
+      description: Signature name to search.
+      isArray: false
+      name: signature_name
+      required: false
+      secret: false
+    - default: false
+      description: Vendor name to search.
+      isArray: false
+      name: vendor
+      required: false
+      secret: false
+    - default: false
+      description: CVE name to search.
+      isArray: false
+      name: cve
+      required: false
+      secret: false
+    - default: false
+      description: From which signature to return results. Default is 0.
+      isArray: false
+      name: from
+      required: false
+      secret: false
+    - default: false
+      description: To which signature to return results. Default is from plus 10.
+      isArray: false
+      name: to
+      required: false
+      secret: false
+    deprecated: false
+    description: Initiate anti spyware signature search.
+    execution: false
+    name: threatvault-antispyware-signature-search
+    outputs:
+    - contextPath: ThreatVault.Search.search_request_id
+      description: Search request ID.
+      type: String
+    - contextPath: ThreatVault.Search.status
+      description: Saearch status.
+      type: String
+  - arguments:
+    - default: false
+      description: Search request ID.
+      isArray: false
+      name: search_request_id
+      required: true
+      secret: false
+    - auto: PREDEFINED
+      default: false
+      description: Search type. ips for antispyware, dns for DNS and panav for antivirus.
+      isArray: false
+      name: search_type
+      predefined:
+      - ips
+      - dns
+      - panav
+      required: true
+      secret: false
+    deprecated: false
+    description: Initiate anti spyware signature search.
+    execution: false
+    name: threatvault-signature-search-results
+    outputs:
+    - contextPath: ThreatVault.Search.search_request_id
+      description: Search request ID.
+      type: String
+    - contextPath: ThreatVault.Search.status
+      description: Search status.
+      type: String
+    - contextPath: ThreatVault.Search.page_count
+      description: How many results returned in this specific search.
+      type: Number
+    - contextPath: ThreatVault.Search.total_count
+      description: How many results are available for this specific search.
+      type: Number
+    - contextPath: ThreatVault.Search.search_type
+      description: Search type. can be wither ips, dns or panav.
+      type: String
+    - contextPath: ThreatVault.Search.signatures
+      description: A list of all the found signatures for this specific search.
+      type: Unknown
+  dockerimage: demisto/python3:3.8.6.12176
+  isfetch: false
+  longRunning: false
+  longRunningPort: false
+  runonce: false
+  script: '-'
+  subtype: python3
+  type: python
+tests:
+- No test
+fromversion: 5.0.0
```

Comment:
need to check if we are should still release for 5.0.0 and not 5.5.0

Reply:
I think we are still at 5.0.0 and above, will validate.